src/queries/text.ts

```diff
@@ -84,11 +84,24 @@ const queryAllByText = (
   options?: TextMatchOptions
 ) => Array<ReactTestInstance>) =>
   function queryAllByTextFn(text, options) {
-    const results = instance.findAll((node) =>
-      getNodeByText(node, text, options)
-    );
+    try {
+      const { Text } = require('react-native');
```

Comment:
Maybe we could extract that `try/catch` as separate function:
```ts
function getTextComponentFromReactNative() {
  try {
    const { Text } = require('react-native');
    return Text;
  } catch (error) {
     throw createLibraryNotSupportedError(error);
  } 
}
```
good place to put it would be `react-native-api.ts` file

This way we would not have to bloat various places with `try/catch/throw` that make the code blocks more visually cluttered.

Reply:
We actually use it only once, the other times are `TextInput`. Should I extract both of them?

Reply:
I think that soon it will be useful also in different places, e.g. for implementing implicit roles for #1130

Reply:
I did extract both. Ideally we should also do it for StyleSheet (which is imported in `isInaccessible`, but that poses some weird types problem, so I'll skip this for now.